.travis.yml

```diff
@@ -31,6 +31,8 @@ before_script:
   # Install composer dependencies
   - composer validate
   - composer require --no-update silverstripe/recipe-core:$RECIPE_VERSION
+  # Fix for running phpunit 5 on php 7.4+
+  - composer require --no-update sminnee/phpunit-mock-objects:^3
```

Comment:
In most other projects we simply add this as a dev dependency.

Reply:
This is a module, not a project